apps/mobile-wallet/src/components/headers/BaseHeader.tsx

```diff
@@ -97,19 +98,21 @@ const BaseHeader = ({
 
   return (
     <BaseHeaderStyled ref={headerRef} onLayout={handleHeaderLayout} {...props}>
-      <HeaderGradientCanvas height={gradientHeight} pointerEvents="none">
-        <Rect x={0} y={0} width={screenWidth} height={gradientHeight} opacity={animatedOpacity}>
-          <LinearGradient
-            start={vec(0, gradientHeight / 1.5)}
-            end={vec(0, gradientHeight)}
-            colors={
-              theme.name === 'dark'
-                ? ['rgba(0, 0, 0, 1)', 'rgba(0, 0, 0, 0)']
-                : ['rgba(255, 255, 255, 1)', 'rgba(255, 255, 255, 0)']
-            }
-          />
-        </Rect>
-      </HeaderGradientCanvas>
+      <View pointerEvents="none">
+        <HeaderGradientCanvas height={gradientHeight} pointerEvents="none">
+          <Rect x={0} y={0} width={screenWidth} height={gradientHeight} opacity={animatedOpacity}>
+            <LinearGradient
+              start={vec(0, gradientHeight / 1.5)}
+              end={vec(0, gradientHeight)}
+              colors={
+                theme.name === 'dark'
+                  ? [theme.bg.back2, colord(theme.bg.back2).alpha(0).toHex()]
+                  : [theme.bg.highlight, colord(theme.bg.highlight).alpha(0).toHex()]
+              }
```

Comment:
I hesitated and decided not to, because I didn't really know where to put it. 
I think that a theme refactor would be nice eventually, and this should probably be a variable in this new theme...

Is it ok if I keep it like that for now?

Reply:
Sure, I don't mind!